build/dart/rules.gni

```diff
@@ -0,0 +1,240 @@
+# Copyright 2015 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file has rules for making Dart packages and Dart-based Mojo applications.
+# The entrypoint is the dart_pkg rule.
+
+import("//build/module_args/mojo.gni")
```

Comment:
It only needs the mojo_sdk_root which is defaulted to //. I can get  rid of all traces of mojo in //build too.